cmake/FindVorbisFile.cmake

```diff
@@ -19,78 +44,103 @@ SET(VORBISFILE_SEARCH_PATHS
 	/opt
 )
 
-SET(MSVC_YEAR_NAME)
-IF (MSVC_VERSION GREATER 1599)		# >= 1600
-	SET(MSVC_YEAR_NAME VS2010)
-ELSEIF(MSVC_VERSION GREATER 1499)	# >= 1500
-	SET(MSVC_YEAR_NAME VS2008)
-ELSEIF(MSVC_VERSION GREATER 1399)	# >= 1400
-	SET(MSVC_YEAR_NAME VS2005)
-ELSEIF(MSVC_VERSION GREATER 1299)	# >= 1300
-	SET(MSVC_YEAR_NAME VS2003)
-ELSEIF(MSVC_VERSION GREATER 1199)	# >= 1200
-	SET(MSVC_YEAR_NAME VS6)
-ENDIF()
-
-FIND_PATH(VORBISFILE_INCLUDE_DIR
+# Map MSVC version to the installation folder name
+set(MSVC_YEAR_NAME)
+if(MSVC_VERSION GREATER 1599)		# >= 1600
+	set(MSVC_YEAR_NAME VS2010)
+elseif(MSVC_VERSION GREATER 1499)	# >= 1500
+	set(MSVC_YEAR_NAME VS2008)
+elseif(MSVC_VERSION GREATER 1399)	# >= 1400
+	set(MSVC_YEAR_NAME VS2005)
+elseif(MSVC_VERSION GREATER 1299)	# >= 1300
+	set(MSVC_YEAR_NAME VS2003)
+elseif(MSVC_VERSION GREATER 1199)	# >= 1200
+	set(MSVC_YEAR_NAME VS6)
+endif()
```

Comment:
MSVC_YEAR_NAME can just be deleted at this point, as can anything mentioning win32 for PATH_SUFFIXES.